analytics/track.go

```diff
@@ -0,0 +1,80 @@
+package analytics
+
+import (
+	"bytes"
+	"encoding/json"
+	"sync"
+	"time"
+
+	"github.com/google/uuid"
```

Comment:
Shouldn't we use the other UUID package we agreed on in the RFC? https://bitrise.atlassian.net/wiki/spaces/ENGI/pages/2122547554/Standardize+used+uuid+package+for+Go+projects#%F0%9F%A4%94-Why?

Reply:
Didn't know about this 😳 will be using this then 👍🏻